docs/source/models/cause.rst

```diff
@@ -275,14 +273,155 @@ Birth Prevalence
+     - Members of the at-risk population die and are no longer susceptible 
+     - Individuals are born or age into the at-risk population and become susceptible
+     - Individuals age out of the at-risk population and are no longer susceptible
+     - Individuals with the condition recover from the condition and re-enter the at-risk population as susceptible (in the case of conditions with remission)
+
+**Total Population Incidence Rate** is estimated by the Global Burden of 
+Disease Study by estimating the number of incident cases that occur in one 
+year and scaling this value per 100,000 individuals of a specified population.
+
+.. math::
+
+  \frac{n_\text{incident cases}}{\text{person-time}_\text{total population}}
+
+Because the denominator of this measure is not specific to a particular cause 
+model state, it is **not** an appropriate data source for cause model 
+transition rates between states. 
+
+.. note:: 
+
+  GBD estimates of total population incidence rate require transformation 
+  prior to use as a cause model transition probability data source (see below 
+  for more detail).
+
+**Susceptible/At-Risk Population Incidence Rate** as discussed here is also 
+referred to as incidence density rate, person-time incidence rate, and in some 
+cases may simply be referred to as the incidence rate. It is defined as:
+
+.. math::
+
+  \frac{n_\text{incident cases}}{\text{person-time}_\text{susceptible population}}
+
+Because the denominator for the susceptible population incidence rate is 
+person-time in the at-risk population, it can represent the probability of a 
+new case of the condition occuring in an individual without the condition in a 
+given time frame. Therefore, it can be used to represent the probability that 
+a simulant will transition from a susceptible to infected cause model state in 
+a given timestep.
+
+  For instance, consider an example in which the global susceptible population 
+  incidence rate of injuries in 2017 was 6,800 cases per 100,000 person-years, 
+  or 0.068 cases per person-year. In this example, 6,800 new injuries occurred 
+  among 100,000 person-years of observation among the non-injured population.
+
+  Now, consider a cause model with a susceptible (not injured) state and an 
+  infected (injured) state with a simulation timestep of 1 year. In this case, 
+  the probability that a simulant will transition from the susceptible to 
+  infected state within a single timestep (i.e. the transition probability) 
+  would be represented as 0.068.
+
+  Notably, in order to represent the transition probability for a single 
+  simulant within a single timestep, the cumulative incidence value needs to 
+  be scaled so that the person-time denominator is equal to the simulation 
+  timestep. Therefore, if the timestep of the cause model considered above 
+  were six months instead of one year, the transition probability would be 
+  0.034 (0.034 cases per 0.5 person-years). 
```

Comment:
As per my comments above, I think it would be good to clarify that these are approximations of the probability, which (I think) are valid because the rate is small enough.

I'm not sure the best way to reword this. Perhaps the current version is good enough for now, and we can think about how to edit it to make it more precise in a future iteration.